### PR TITLE
Add COPYING file name as valid license file

### PR DESCRIPTION
**Pull request by yland**
**What this PR does / why we need it**:
Some valuable libraries such as https://github.com/google/shlex which could be integrated into Kubernetes use COPYING for their License file name. This would be helpful in getting pull requests such as https://github.com/kubernetes/kubernetes/pull/59700 merged.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: hack/update-godep-licenses.sh [hunk 1]
```diff
@@ -171,6 +176,8 @@ for PACKAGE in $(cat Godeps/Godeps.json | \
       file="${CONTENT[${PACKAGE}-LICENSE]-}"
   elif [[ -n "${CONTENT[${PACKAGE}-COPYRIGHT]-}" ]]; then
       file="${CONTENT[${PACKAGE}-COPYRIGHT]-}"
+  elif [[ -n "${CONTENT[${PACKAGE}-COPYING]-}" ]]; then
+      file="${CONTENT[${PACKAGE}-COPYING]-}"
   fi
   if [[ -z "${file}" ]]; then
       cat > /dev/stderr << __EOF__
```
Comment (ixdy) [inline hack/update-godep-licenses.sh]: maybe also update the message on L187-190 to mention COPYING?

**Comment by ixdy** [comment]
besides that minor suggestion, lgtm

File: hack/update-godep-licenses.sh (hunk 1 above)
Comment (yland) [reply hack/update-godep-licenses.sh]: @ixdy Thanks for the review, I have made the suggested updates.

**Pull request merged**